### DISCUSSION TITLE
Is there a plugin for MSI boards to control cpu_fan header with FanControl?

### DISCUSSION THREAD

**Discussion by Dieter-Kahl** [Rem0o/FanControl.Releases#2754] [Q&A]
Sorry if there is, I've always been bad at navigating github and even worse at finding issues that have already been addressed.

**Comment by AngelOfMors** [comment]
Why do you need a separate plugin if the program sees this connector?

**Comment by Dieter-Kahl** [comment]
The program doesn't recognize this connector for me, and after reading
around a bit, I thought that was the case for everybody. I saw that a
separate make of boards (Asus?) has a plugin to address this, so I wondered
if MSI does as well.

**Comment by AngelOfMors** [comment]
Then you better indicate which motherboard.

**Comment by Dieter-Kahl** [comment]
MSI MPG Z690 Edge WIFI (DDR4)

**Comment by AngelOfMors** [comment]
MSI MPG Z790 Edge WiFi (DDR5), Win11
Normal

**Comment by Dieter-Kahl** [comment]
MSI MPG Z690 Edge WiFi (DDR4), win 10

**Comment by Rem0o** [comment]
There is no such plugin .
